src/datafile.rkt

```diff
@@ -29,27 +29,27 @@
                note
                tests))
 
-(define (try-list-accessor acc fail)
-  (λ (l) (if (null? l) fail (acc l))))
```

Comment:
So you can run herbie with `--no-pareto` and in that case, the `cost-accuracy` list will be `null?`, which is what this code was supposed to catch. Can we still make that work?